src/views/dialogs/DetailsDialog/OrderDetailsDialog.tsx

```diff
@@ -100,7 +98,7 @@ export const OrderDetailsDialog = ({
     );
 
   const renderOrderTime = ({ timeInMs }: { timeInMs: Nullable<number> }) =>
-    timeInMs ? <time>{relativeTimeString({ timeInMs, selectedLocale })}</time> : '-';
+    timeInMs ? <Output type={OutputType.DateTime} value={timeInMs} /> : '-';
```

Comment:
I don't think we need to use a ternary here since the `<Output>` component handles `timeInMs = null` by rendering `-`

Reply:
As far as I can tell that is only true for numbers. Dates and relative dates bail early and return null if things aren't quite right I think. 

We can make dates behave similarly, I'd just do a quick audit to make sure it doesn't break any existing behavior.